### PR TITLE
media-sound/audacious: Version bump

### PR DESCRIPTION
**Pull request by RobinDX**
Both audacious and audacious-plugins version bump to 3.7.2

### CODE REVIEW

File: media-plugins/audacious-plugins/audacious-plugins-3.7.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,173 @@
+# Copyright 1999-2015 Gentoo Foundation
```
Comment (monsieurp) [inline media-plugins/audacious-plugins/audacious-plugins-3.7.2.ebuild]: Please change this to 2016.

File: media-plugins/audacious-plugins/audacious-plugins-3.7.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```
Comment (monsieurp) [inline media-plugins/audacious-plugins/audacious-plugins-3.7.2.ebuild]: EAPI 6?

File: media-sound/audacious/audacious-3.7.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2015 Gentoo Foundation
```
Comment (monsieurp) [inline media-sound/audacious/audacious-3.7.2.ebuild]: Same here, this should be changed to 2016.

File: media-sound/audacious/audacious-3.7.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```
Comment (monsieurp) [inline media-sound/audacious/audacious-3.7.2.ebuild]: EAPI 6 please.

**Comment by monsieurp** [comment]
@gentoochainsaw

File: media-sound/audacious/audacious-3.7.2.ebuild (hunk 4 above)
Comment (mjeveritt) [inline media-sound/audacious/audacious-3.7.2.ebuild]: someone hasn't run repoman?!

**Comment by monsieurp** [comment]
Please make sure your commits comply with our standards.

**Comment by monsieurp** [comment]
ping @RobinDX?

**Comment by monsieurp** [comment]
Reopen when commits are fixed.

**Pull request closed without merging**